config/initializers/event_bus_mailers.rb

```diff
@@ -0,0 +1,9 @@
+
+EventBus.listen('new_comment_event') do |event|
```

Comment:
No, there's two different events - one that's emitted after the comment is
created, one that's created after the new_comment event is created. We need
the event in this case because the email uses @event to generate the utm
hash for the links at the bottom (and maybe some other stuff)

On Fri, Dec 18, 2015 at 2:57 AM, Robert Guthrie <notifications@github.com>
wrote:

> In config/initializers/event_bus_mailers.rb
> <https://github.com/loomio/loomio/pull/2804#discussion_r47986874>:
>
> > @@ -0,0 +1,9 @@
> > +
> > +EventBus.listen('new_comment_event') do |event|
>
> is this supposed to be new_comment (without _event)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2804/files#r47986874>.
>